Gemfile.jruby-2.5.lock.release

```diff
@@ -908,18 +908,16 @@ DEPENDENCIES
   logstash-output-tcp
   logstash-output-udp
   logstash-output-webhdfs
-  octokit (~> 4)
+  octokit (~> 4.25)
   paquet (~> 0.2)
   pleaserun (~> 0.0.28)
-  polyglot
```

Comment:
This and `treetop` only seemed to come in in `8.3.0` after the [polyglot removal](https://github.com/elastic/logstash/commit/6036b3d14108fed730b642654d26413282f72504), but did not go away after that was [reverted](https://github.com/elastic/logstash/pull/14279)  https://github.com/elastic/logstash/blob/v8.2.3/Gemfile.jruby-2.5.lock.release